webui/master/src/containers/pages/Mounttable/Mounttable.tsx

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
for consistency, can you rename this file to `MountTable.tsx`

Reply:
Yeah.